src/dev/jest/config.js

```diff
@@ -49,6 +55,30 @@ export default {
     '<rootDir>/src/test_utils',
     '<rootDir>/test/functional/services/remote',
   ],
+];
+
+const roots = [];
+
+// Looks for --ci-group=<number> and captures the number
+const ciGroupPattern = /^--ci-group=(\d+)$/;
+const ciGroup = process.argv.reduce((acc, arg) => {
```

Comment:
Shouldn't we document this new flag somewhere? I'm not sure there's a way to add it to the jest `--help` menu, but maybe on `TESTING.md`?

Reply:
i will add it.